### PR TITLE
Add cases for libvirt_package

### PR DESCRIPTION
**Pull request by cliping**
Signed-off-by: Liping Cheng <lcheng@redhat.com>

### CODE REVIEW

**Comment by cliping** [comment]
Test result:

#avocado run --vt-type libvirt --vt-machine-type q35 libvirt_package --vt-connect-uri qemu:///system
/root/.libvirt-ci-venv-ci-runtest-ZYrh5X/lib/python3.6/site-packages/avocado_framework-80.0-py3.6.egg/avocado/plugins/run.py:292: FutureWarning: The following arguments will be changed to boolean soon: sysinfo, output-check, failfast and keep-tmp. 
  FutureWarning)
JOB ID     : 4713b52083ea94dd1897cf04efae044e8758e370
JOB LOG    : /root/avocado/job-results/job-2020-08-04T06.07-4713b52/job.log
 (1/4) type_specific.io-github-autotest-libvirt.libvirt_package.libvirtd_start.lock_sanlock: PASS (9.26 s)
 (2/4) type_specific.io-github-autotest-libvirt.libvirt_package.libvirtd_start.daemon_driver_qemu: PASS (9.34 s)
 (3/4) type_specific.io-github-autotest-libvirt.libvirt_package.libvirtd_stop.lock_sanlock: PASS (8.95 s)
 (4/4) type_specific.io-github-autotest-libvirt.libvirt_package.libvirtd_stop.daemon_driver_qemu: PASS (9.55 s)
RESULTS    : PASS 4 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 39.60 s

File: libvirt/tests/src/libvirt_package.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        pkg_mgr.install()
+        # verify package
+        cmd = "rpm -V %s" % pkg_name
+        ret = process.run("rpm -V %s" % pkg_name, shell=True)
+        if ret.exit_status:
+            test.fail("Check %s failed." % pkg_name)
+        if ret.stdout_text.strip():
+            test.fail("Verify %s failed: %s" % (pkg_name, ret.stdout_text.strip()))
+    finally:
+        vmxml_backup.sync()
```
Comment (Yingshun) [inline libvirt/tests/src/libvirt_package.py:47]: I think it'll be great if we remove the installed package in 'finally' block. How do you think?

Comment (cliping) [reply libvirt/tests/src/libvirt_package.py:47]: If we remove the installed package, maybe lead other cases which need the packages failed.

**Comment by cliping** [comment]
Test result:

# avocado run --vt-type libvirt --vt-machine-type q35 libvirt_package --vt-connect-uri qemu:///system
/root/.libvirt-ci-venv-ci-runtest-ZYrh5X/lib/python3.6/site-packages/avocado_framework-80.0-py3.6.egg/avocado/plugins/run.py:292: FutureWarning: The following arguments will be changed to boolean soon: sysinfo, output-check, failfast and keep-tmp. 
  FutureWarning)
JOB ID     : f96f133ebc6e48f9dfa59d2f26dfa03df1456a17
JOB LOG    : /root/avocado/job-results/job-2020-08-07T01.02-f96f133/job.log
 (1/6) type_specific.io-github-autotest-libvirt.libvirt_package.libvirtd_start.lock_sanlock: PASS (6.86 s)
 (2/6) type_specific.io-github-autotest-libvirt.libvirt_package.libvirtd_start.daemon_driver_qemu: PASS (9.41 s)
 (3/6) type_specific.io-github-autotest-libvirt.libvirt_package.libvirtd_start.daemon_config_nwfilter: PASS (7.63 s)
 (4/6) type_specific.io-github-autotest-libvirt.libvirt_package.libvirtd_stop.lock_sanlock: PASS (7.57 s)
 (5/6) type_specific.io-github-autotest-libvirt.libvirt_package.libvirtd_stop.daemon_driver_qemu: PASS (9.56 s)
 (6/6) type_specific.io-github-autotest-libvirt.libvirt_package.libvirtd_stop.daemon_config_nwfilter: PASS (8.17 s)
RESULTS    : PASS 6 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 51.77 s

**Comment by dzhengfy** [comment]
@cliping , please remember to add a comment if your pr is to test a customer bug. Then the maintainer will add the label. Thanks.

**Comment by cliping** [comment]
> @cliping , please remember to add a comment if your pr is to test a customer bug. Then the maintainer will add the label. Thanks.

Ok.

File: libvirt/tests/cfg/libvirt_package.cfg [hunk 2]
```diff
@@ -0,0 +1,15 @@
+- libvirt_package:
+    type = libvirt_package
+    start_vm = 'no'
+    variants:
+        - lock_sanlock:
```
Comment (chunfuwen) [inline libvirt/tests/cfg/libvirt_package.cfg:5]: is it good way to consider merge those variant as one, and just install the packages one by one in one case?

Comment (cliping) [reply libvirt/tests/cfg/libvirt_package.cfg]: Updated.

**Comment by cliping** [comment]
Test result:

#avocado run --vt-type libvirt --vt-machine-type q35 libvirt_package --vt-connect-uri qemu:///system
JOB ID     : b5743f6a8c53aad5ab41009ea119352f5bb3e5a9
JOB LOG    : /root/avocado/job-results/job-2020-08-18T11.06-b5743f6/job.log
 (1/2) type_specific.io-github-autotest-libvirt.libvirt_package.libvirtd_start: PASS (32.81 s)
 (2/2) type_specific.io-github-autotest-libvirt.libvirt_package.libvirtd_stop: PASS (43.82 s)
RESULTS    : PASS 2 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 78.91 s

**Review by chunfuwen — APPROVED** [review]

**Pull request merged**